hacker-news-app/src/Components/Backdrop/Backdrop.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+
+const Display = styled.div`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.3);
+  z-index: 100;
+  top: 0;
+  left: 0;
+`;
+
+function Backdrop(props) {
+    
+  return <Display className="backdrop" onClick={()=>props.toggleButton()}></Display>;
```

Comment:
Is that `className` being used anywhere? If not 👉 🗑 👈

Reply:
I will remove this and raise it with the other front-end devs.